targets/TARGET_WICED/WICED.md

```diff
@@ -0,0 +1,229 @@
+# How to add the WICED SDK
+
+This document summarizes how to download and integrate the WICED SDK into Mbed OS.
+
+**NOTE:** This document is intended for when the WICED SDK is extracted from Mbed OS.
+
+There are several files not provided by the WICED SDK. This is a non-exhaustive
+list of files that are currently provided by Mbed OS:
+- w_generated_mac_address.txt
+- w_generated_security_dct.h
+- w_resources.h
+- mbed_lib.json
+
+## Download the WICED SDK
+
```

Comment:
Here are two different instructions, so I would add comment:
"Both WICED SDK and patch from Advantech is needed"